docs/quarantine.md

```diff
@@ -102,3 +102,25 @@ After merging this PR the test will be out of quarantine.
 [2]: https://www.thoughtworks.com/en-us/insights/blog/no-more-flaky-tests-go-team
 [3]: https://docs.gitlab.com/ee/development/testing_guide/flaky_tests.html#quarantined-tests
 [on testgrid]: https://testgrid.k8s.io/kubevirt-periodics
+
+# Test Lane Quarantine
+
+There can be cases where required test lanes are flaking in a clustered manner with a large
+number of tests failing each time. This can cause major delays to merging important pull
+requests and overload CI. Individual test case quarantining does not make sense in this case.
+
+A required test lane should be made optional if the following criteria are met:
+* The required test lane has a failure rate higher than 40% in the last 24 hours
+* More than ten individual test cases are causing the required test lane to fail
+* The SIG responsible for the required test lane is unable to deliver a fix for the
+flake within 24 hours
+
+Following a required test lane being made optional a number of actions must happen:
+* Create github issue with a comment `/release-blocker main` to ensure that
+the issue is addressed before a new release is cut.
+* Ensure that the github issue is assigned to a member of the responsible SIG who
+will own bringing the blocker to completion within a quick time frame.
+* The SIG should stop all feature and refactoring work until a fix for the flake
```

Comment:
Can we add an enforcement action here? Can be a subsection.
How exactly this is enforced? Even if it is not automated/configured, the responsibility should be explicitly defined on the one who us responsible to assure no PRs are merged unless it is fixing the flake (e.g. it could be under the specific SIG responsibility).

Reply:
I was thinking about how this could be enforced - SIG CI could hold PRs if they are not happy that the flakes are being given the appropriate focus by the responsible SIG. 

Something like:
```
If the flaky test lane is not receiving the required attention from the responsible SIG, 
SIG CI will hold merge queue PRs from the responsible SIG that are not related to a fix.
```

Reply:
IMO the decision should be of SIG-CI or the test owners, but the implementation should be on the SIG that owns the tests. If they fail, only then the SIG-CI should take an extreme action and start actively blocking those PRs. If the SIG-CI take a clocking action, it is an extreme and I would consider the SIG owning the tests as not confirming to the policy.